src/modules/transaction/utils/transaction.js

```diff
@@ -101,6 +105,47 @@ const getTransactionAmount = ({ module, command, params = {} }) => {
   return '0';
 };
 
+export const getTransactionValue = (transactionJSON, token) => {
+  const { params, moduleCommand, id } = transactionJSON;
+
+  const getTokenWithSymbol = (tokenInBaseDenom) => (
+    <TokenAmount val={tokenInBaseDenom} token={token} />
+  );
+
+  const genrateLink = (text, href) => (
+    <Link to={href} onClick={(e) => e.stopPropagation()}>
+      {text}
+    </Link>
+  );
+
+  const trasnactionValueMap = {
+    [MODULE_COMMANDS_NAME_MAP.changeCommission]: () => `${+params.newCommission / 100}%`,
+    [MODULE_COMMANDS_NAME_MAP.claimRewards]: () => null,
+    [MODULE_COMMANDS_NAME_MAP.unlock]: () => null,
```

Comment:
For unlock and claim rewards we can use events tab? which has data? similar to `Signatories` logic you can provide a link for events? 

i.e instead of `-` for these trx we can use `events` and make that as a link to redirect to events tab next to `Details`?

Reply:
Ignore this suggestion.